tests/unit/s2n_x509_validator_test.c

```diff
@@ -437,7 +444,8 @@ int main(int argc, char **argv) {
         EXPECT_SUCCESS(s2n_pkey_zero_init(&public_key_out));
         s2n_pkey_type pkey_type = S2N_PKEY_TYPE_UNKNOWN;
         EXPECT_ERROR_WITH_ERRNO(s2n_x509_validator_validate_cert_chain(&validator, connection, chain_data, chain_len,
-                &pkey_type, &public_key_out), S2N_ERR_CERT_EXPIRED);
+                                        &pkey_type, &public_key_out),
+                S2N_ERR_CERT_EXPIRED);
```

Comment:
odd formatting due to nested fn calls. is there a way to make this nicer?

Reply:
There are a wide range of options that clang-format accepts for formatting something like this. In a editor with a integrated clang-format I don't think this will be much of a problem, since things will be formatted as you write them. In a large scale change like this this kind of thing comes up a lot.

With the way we've configured it it very often gets this right. Its these long nested function calls that it doesn't do very well with.


First lets take a look at the one argument case: We choose a clang-format option that wraps function calls with many arguments as follows:
```c
EXPECT_ERROR_WITH_ERRNO(s2n_x509_validator_validate_cert_chain(&validator, connection,
                chain_data, chain_len, &pkey_type, &public_key_out));
```
The argument follows on the next line with a predictable indent. When we add a second argument we apply the same rules:
## Option 0 (no change)
```c
EXPECT_ERROR_WITH_ERRNO(s2n_x509_validator_validate_cert_chain(&validator, connection,
                                        chain_data, chain_len, &pkey_type, &public_key_out),
                S2N_ERR_CERT_EXPIRED);
```
You can see `s2n_x509_validator_validate_cert_chain` has been flowed in the exact same manner -- this time indented from the start of `s2n_x509_validator_validate_cert_chain`.

Even though this is a consistent application of the style rule, its not pleasing to the eye.

But we have options first we can remove all the new line:

## Option 1:
```c
        EXPECT_ERROR_WITH_ERRNO(s2n_x509_validator_validate_cert_chain(&validator, connection, chain_data, chain_len, &pkey_type, &public_key_out), S2N_ERR_CERT_EXPIRED);
```

Clang-format will accept this but we can't -- the line is simply too long. This answers your previous comment -- why don't we enforce a line length? Because in this and other scenarios we would prefer to select where newlines get inserted.

So lets insert a newline between the 2 arguments:

## Option 2:
```c
        EXPECT_ERROR_WITH_ERRNO(s2n_x509_validator_validate_cert_chain(&validator, connection, chain_data, chain_len, &pkey_type, &public_key_out),
                S2N_ERR_CERT_EXPIRED);
```

Okay the line is slightly shorter. But still too long. We can't break up the `s2n_x509_validator_validate_cert_chain` arguments or we will be in the situation above. So we break before the function call:

## Option 3:
```c
EXPECT_ERROR_WITH_ERRNO(
                s2n_x509_validator_validate_cert_chain(&validator, connection, chain_data, chain_len, &pkey_type, &public_key_out),
                S2N_ERR_CERT_EXPIRED);
```

This may or may not be acceptable. But doing this will allow us to more reasonably insert a newline in the inner argument list:
## Option 4:
```c
        EXPECT_ERROR_WITH_ERRNO(
                s2n_x509_validator_validate_cert_chain(&validator, connection, chain_data,
                        chain_len, &pkey_type, &public_key_out),
                S2N_ERR_CERT_EXPIRED);
```

Let me know what you think. Which option do you prefer?

Reply:
I've pushed a change with 2 so you can see it in context.

Reply:
4 looks good to me simply based on readibility